### PR TITLE
Move more operators to use 4.12 compact cluster

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
Depends-On: https://github.com/openshift/release/pull/41385

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-heat-operator-main-heat-operator-build-deploy-kuttl

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ovn-operator-main-ovn-operator-build-deploy-kuttl pull-ci-openstack-k8s-operators-horizon-operator-main-horizon-operator-build-deploy-kuttl pull-ci-openstack-k8s-operators-infra-operator-main-infra-operator-build-deploy

File: ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml [hunk 1]
```diff
@@ -77,9 +77,11 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h0m0s
-    version: "4.10"
+    version: "4.12"
```
Comment (bshephar) [inline ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml:80]: So this version mentioned in this file references the install-config.yaml here:
https://github.com/openshift/release/blob/master/clusters/hive/pools/openstack-k8s-operators/install-config-hybrid-3cp_secret.yaml

Is my understanding correct?

**Review by bshephar — APPROVED** [review]
Looks fine to me. Just a question of how the pieces come together.

File: ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml (hunk 1 above)
Comment (viroel) [reply ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml:80]: @bshephar yes, this is the secret `install-config-hybrid-3cp` that we reference in this cluster pool: https://github.com/openshift/release/blob/master/clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml#L22
which has the label `version: "4.12"` used by the CI to select the correct pool.

**Comment by viroel** [comment]
moving more cluster from 4.10 to 4.12 pool here https://github.com/openshift/release/pull/41587

**Pull request merged**